cookbook/bundles/best_practices.rst

```diff
@@ -102,9 +101,8 @@ that automated tools can rely on:
 * ``README.md``: This file contains the basic description of the bundle and it
   usually shows some basic examples and links to its full documentation (it
   can use any of the markup formats supported by GitHub, such as ``README.rst``);
-* ``Resources/meta/LICENSE``: The full license for the code. The license file
-  can also be stored in the bundle's root directory to follow the generic
-  conventions about packages;
+* ``LICENSE``: The full license for the code. This license file can also be stored
+  in the bundle's ``Resources/meta/`` directory for backwards compatibility reasons;
```

Comment:
is it possible to add a symlink for backward compatibility?

then i recommend this to avoid different files